crates/core/src/client/constants.rs

```diff
@@ -5,6 +5,9 @@ use starknet::core::types::FieldElement;
 /// CHAIN_ID = KKRT (0x4b4b5254) in ASCII
 pub const CHAIN_ID: u64 = 1_263_227_476;
 
+// TODO: Automate changing this version as we do future releases
```

Comment:
Added a todo, so that in future we can automate this &str being updates as we bump new versions.

Reply:
It should probably take the version from `Cargo.toml` of the main RPC workspace.